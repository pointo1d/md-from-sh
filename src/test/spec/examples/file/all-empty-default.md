# all-empty-default.sh

---

# Description

---

# Synopsis

---

# Where

---

# Opts

---

# Args

---

# Returns

---

# Env Vars

---

# Notes

---

# Functions

---

# Doc Links

---

# Files

---

# To Do

---

# Author

---

# Date

---

# Copyright

---

# License

---

END OF FILE
