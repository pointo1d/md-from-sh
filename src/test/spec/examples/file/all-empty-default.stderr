WARNING - Empty section: 'File' - line 4
WARNING - Empty section: 'Title' - line 6
WARNING - Empty section: 'Synopsis' - line 7
WARNING - Empty section: 'Description' - line 8
WARNING - Empty section: 'Opts' - line 9
WARNING - Empty section: 'Args' - line 10
WARNING - Empty section: 'Env Vars' - line 11
WARNING - Empty section: 'Notes' - line 12
WARNING - Empty section: 'To Do' - line 13
WARNING - Empty section: 'Files' - line 14
WARNING - Empty section: 'Author' - line 16
WARNING - Empty section: 'Date' - line 17
WARNING - Empty markdown
